### DISCUSSION TITLE
Update to German

### DISCUSSION THREAD

**Discussion by petzi69** [aclap-dev/video-downloadhelper#821] [Language & Translation]
[de.txt](https://github.com/user-attachments/files/15573566/de.txt)

**Comment by wjust62** [comment]
improved
[de.txt](https://github.com/user-attachments/files/15676164/de.txt)

**Comment by Nictrical** [comment]
Here goes my version (no grammar errors or typos):

[de.txt](https://github.com/user-attachments/files/15748591/de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
